mod-login-saml/src/main/resources/domain/mod-login-saml/features/loginSaml.feature

```diff
@@ -3,11 +3,55 @@ Feature: Login SAML
   Background:
     * url baseUrl
     * callonce login testUser
-    * configure headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/json, text/plain' }
-
-  Scenario: SAML Check
-     Given path 'saml/check'
-     And header x-okapi-tenant = testTenant
-     And header x-okapi-token = '#(okapitoken)'
-     When method GET
-     Then status 200
+    * configure headers = { 'Content-Type': 'application/json', 'x-okapi-tenant': #(testTenant), 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/json, text/plain' }
+
+  Scenario: Check endpoint missing okapi url header active is false
```

Comment:
Do not think we need to mention `missing okapi url header`. For Karate integration test, the call goes to Okapi directly, so Okapi url header is not needed. This test scenario is essentially `Check endpoint returns active is false when IdP is not configured`

Reply:
I agree. I think your wording of it is better. Let me make that change and I'll go ahead and merge.